src/BL/WeatherForecast.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BL
+{
+    public class WeatherForecast
+    {
+        private static string apiKey = "c1c4d772f711221a4a1df5be101bb4a5";
+        private static string url1 = "https://api.openweathermap.org/data/2.5/weather?q=";
+        //private static string city = String.Empty;
+        private static string url2 = "&appid=";
+        private static string units = "&units=metric";
+        public static async Task<float> GetTemperature(string city)
```

Comment:
It is common to use `double` for (ironically) floating numbers in C#. Smaller size `float` does not really beat `double` [performance-wise](https://stackoverflow.com/questions/417568/float-vs-double-performance). Another reason for using `double` is laziness. For double literals u don't have to use any suffix unlike in `float` u have to use `-f/F`. 

If you type `var r = 1.2`, the compiler shows that `r` is a `double` type. Why not go with `double`, right.

Reply:
Changed float? to double?